Adyen/Assets/pt-BR.lproj/Localizable.strings

```diff
@@ -86,6 +86,7 @@
 "adyen.billingAddressSection.title" = "Endereço de cobrança";
 "adyen.billingAddress.placeholder" = "Endereço de cobrança";
```

Comment:
```suggestion
"adyen.billingAddress.placeholder" = "Seu endereço de cobrança";
```

Reply:
Thanks for the suggestion - will fix it in a subsequent PR